### PR TITLE
Add booleans concept

### PR DESCRIPTION
**Pull request by pwadsworth**
Adds Booleans concept ported from F#.

### CODE REVIEW

File: concepts/booleans/about.md
Comment (ErikSchierboom) [inline concepts/booleans/about.md:10]: We probably want this section in the `introduction.md` too, as student will have to know about it in order to successfully finish the corresponding exercise.

**Comment by kytrinyx** [comment]
This is very timely. I just started a draft PR to start implementing the concept exercise that introduces booleans. It's great to have the work on the concept itself being done now as well. Thank you for that!

**Comment by pwadsworth** [comment]
> This is very timely. I just started a draft PR to start implementing the concept exercise that introduces booleans. It's great to have the work on the concept itself being done now as well. Thank you for that!

@kytrinyx how far along the exercise are you? I was working Annalyn's Infiltration too. Just have a few more tests to port.

File: concepts/booleans/about.md
Comment (pwadsworth) [reply concepts/booleans/about.md:10]: Done.

Comment (kytrinyx) [inline concepts/booleans/about.md:10]: I would say "override" rather than "bypass" in this case.

A bypass is a road that passes around something, such as a residential area while an override is a mechanism, device or procedure used to counteract an automatic control.

Comment (pwadsworth) [reply concepts/booleans/about.md]: Edited introduction.md and about.md to reflect suggested change.

File: concepts/booleans/about.md
Comment (kytrinyx) [inline concepts/booleans/about.md:10]: An extra **on** slipped into this by accident, it looks like. It should be `If you want to override these rules`.

Comment (pwadsworth) [reply concepts/booleans/about.md]: Edited. Thanks.

**Comment by kytrinyx** [comment]
This looks good to me, now. @pwadsworth did you decide that you don't want to submit this change afterall?

**Comment by pwadsworth** [comment]
Sorry, I closed the fork by accident 😳.

**Comment by kytrinyx** [comment]
> Sorry, I closed the fork by accident 😳.

I'm so relieved to hear that 😂 I was worried that I had driven you off with all the chaos!

**Comment by pwadsworth** [comment]
Is this PR pending some action or just waiting for the exercise to be merged?

**Review by kytrinyx — APPROVED** [review]

**Comment by kytrinyx** [comment]
No, this just got lost off my list in the shuffle. Merging now.

**Comment by petertseng** [comment]
These commits should have been squashed before merging. Let's all do our part to keep the history of this repository understandable in order to help our future selves when our future selves need to understand changes made by our past selves.

**Comment by kytrinyx** [comment]
I'm sorry, @petertseng. There are different rules and preferences in the various repos, and I didn't think to check here. I will do better next time.

**Comment by petertseng** [comment]
I must be making life difficult because of my not having documented the preferences here. Although like most other repos this repo has a README, the README has always been geared toward contributors and not toward maintainers since contributors are more numerous. I wonder if there is a place that such a preference should be documented, to make things better.

**Comment by kytrinyx** [comment]
@petertseng I've asked Jeremy and Erik if we have anything for this. For now I will make a note that it's something that we will need to figure out going forward.

**Pull request merged**
